File: backend/src/main/java/com/example/template/dtos/LibroGetByIdResponseDTO.java
package com.example.template.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.*;
import com.example.template.enums.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LibroGetByIdResponseDTO {
private long LibroId;
private String Nombre;
private double Precio;
private Genero Genero;
private AutorGetByIdResponseDTO Autor;
}

File: frontend/src/dtos/Libro/getById/LibroGetByIdResponseDto.ts
import AutorGetByIdResponseDto from "../../Autor/getById/AutorGetByIdResponseDto";import { Genero } from "../../../enums/Genero";
export default interface LibroGetByIdResponseDto
{
	libroId : number,
	nombre : string
	precio : number
	genero : Genero

	autor : AutorGetByIdResponseDto;

}
